components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml

```diff
@@ -2427,6 +2427,69 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+######################################################
+# The "API Throttling Policy" resource APIs
+######################################################
+  '/throttling-policies/subscription/api/{apiId}':
```

Comment:
Shall we rename this to `/apis/{apiId}/subscription-policies`.

Also, it is better to move the API upwards adjacent to set of APIs which are under `apis/{apiId}`

Reply:
Fixed